components/ledger/.env.example

```
@@ -142,11 +159,75 @@ RABBITMQ_CONSUMER_PASS=lerian
 RABBITMQ_NUMBERS_OF_WORKERS=5
 RABBITMQ_NUMBERS_OF_PREFETCH=10
 RABBITMQ_HEALTH_CHECK_URL=http://${RABBITMQ_HOST}:${RABBITMQ_PORT_AMQP}
-RABBITMQ_BALANCE_CREATE_QUEUE=transaction.balance_create.queue
 RABBITMQ_TRANSACTION_BALANCE_OPERATION_EXCHANGE=transaction.transaction_balance_operation.exchange
 RABBITMQ_TRANSACTION_BALANCE_OPERATION_KEY=transaction.transaction_balance_operation.key
 RABBITMQ_TRANSACTION_BALANCE_OPERATION_QUEUE=transaction.transaction_balance_operation.queue
 RABBITMQ_TRANSACTION_ASYNC=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_TRANSACTION_ASYNC key should go before the RABBITMQ_TRANSACTION_BALANCE_OPERATION_EXCHANGE key

(UnorderedKey)